src/hatch/template/files_feature_ci.py

```diff
@@ -38,9 +38,24 @@ class CommandLinePackage(File):
       with:
         python-version: ${{ matrix.python-version }}
 
+    - uses: actions/cache@v3
```

Comment:
`actions/setup-python@v4` above supports some level of caching — would it be enough to just enable it there?

Reply:
I don't think it will be relevant, because the python action caching caches the dependencies of python (those that you pip install), but here I keep the virtual environment that was created by hatch because this is the one that takes time to initialize.

In another note, it may be a good thing to have an option to install dependencies without venv, but this is for another pr

Reply:
Then you definitely need a better cache key...